components/docs-chef-io/content/automate/ha_upgrade_introduction.md

```diff
@@ -79,4 +79,105 @@ We can also pass a flag in upgade command to avoid prompt for workspace upgrade.
    chef-automate upgrade run --airgap-bundle latest.aib --auto-approve --workspace-upgrade no
   ```
 
+{{< note >}}
+
+  AMI Upgrade is only for AWS deployment ,as in Onpremise Deployment all the resources are managed by customers its self.  
+{{< /note >}}
+
+## AMI Upgrade Setup For AWS Deployment
+
+### Requirements
+
+1. Two identical clusters located in same/different AWS regions.
+1. Amazon S3 access in both regions for Application backup.
+
+In the above approach, there will be two identical clusters
+
+- Primary Cluster (or Production Cluster)
+- Secondary Cluster (Having upgraded AMI)
+
+{{< note >}}
+
+  The AWS deployment should be configured with S3.  
+{{< /note >}}
+
+### Steps to set up the AMI Upgraded Cluster
+
+1. Deploy the Primary cluster following the deployment instructions by [clicking here](/automate/ha_aws_deploy_steps/#deployment).
```

Comment:
The assumption here is the user will already have a Primary setup (in case of an AMI upgrade)

Should we use the terminology `Primary/Secondary cluster`?

Reply:
As Discussed ,used Primary and New cluster